library/connect.c

```diff
@@ -603,20 +605,6 @@ static void ziti_write_async(uv_async_t *ar) {
             send_message(conn, ContentTypeData, req->buf, req->len, req);
         }
     }
-    uv_close((uv_handle_t *) ar, free_handle);
-}
-
-int ziti_write_req(struct ziti_write_req_s *req) {
-    NEWP(ar, uv_async_t);
-    uv_async_init(req->conn->ziti_ctx->loop, ar, ziti_write_async);
```

Comment:
this code is being removed, but I think `uv_async_init` should have been serialized if being called from another thread, since it modifies the loop?

Reply:
yes, it was a bad pattern